docker-compose.yml

```diff
@@ -563,9 +567,9 @@ services:
       - |
         set -e
 
-        # Add static route to app subnet via router
-        ip route add 172.28.0.0/24 via 172.29.0.254
-        ip -6 route add 172:28:0::/64 via 172:29:0::254
+        # Add static route to internet subnet via router
+        ip -4 route add 200.0.0.0/8 via 172.29.0.254
```

Comment:
Since IPv4 address space is much smaller, these might collide with publicly routable IPs - thoughts on using one of the [Test-Net-3 blocks](https://en.wikipedia.org/wiki/Reserved_IP_addresses)?

Reply:
Probably a good idea, thanks!